### ISSUE TITLE
Credit Agricol incorrect parsing of "label" and "date" in operations

### ISSUE THREAD

**Issue by francoisWeber** [cozy/cozy-konnector-bank-cragr#6]
It seems the Credit Agricole connector does not always separate well the "label" and "date" fields that are written in io.cozy.bank.operations.
Example : 
{amount : -9.6
currency : "EUR"
date : "2017-07-24T00:00:00.000Z"
dateImport : "2017-12-15T10:53:42.077Z"
dateOperation : "24-jul"
label : **"Flunch Plaisir             22/07"**}

Occurs in ~66% of known operations

**Comment by doubleface** [comment]
Yes, in this case 22/07 is the date in which the operation really happened and this is not parsed, yet.
